native/CMakeLists.txt

```diff
@@ -2254,6 +2211,15 @@ else()
     )
 endif()
 
+source_group(TREE ${CWD} PREFIX "Source Files" FILES ${COCOS2D_SOURCE_LIST})
+
+if(QNX)
+    include(${QNX_PATH}/source.cmake)
+    qnx_source_list(ccqnx_SOURCE_LIST ${QNX_PATH})
+    source_group(TREE ${QNX_PATH} PREFIX "Source Files" FILES ${ccqnx_SOURCE_LIST}})
+    list(APPEND COCOS2D_SOURCE_LIST ${ccqnx_SOURCE_LIST})
```

Comment:
Add the source file path to the ccqnx module to avoid using external parameters in qnx

Reply:
How about rename ccqnx to CCQNX as other words are capital.

Reply:
Ok.